curation_concerns-models/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+
+# Specify your gem's dependencies in sufia-models.gemspec
```

Comment:
There will be a lot of these...I figured I'd just grep for comments and replace them with sed later